tests/foreman/ui/test_hostcollection.py

```diff
@@ -390,7 +390,7 @@ def test_negative_hosts_limit(self):
         cv.publish()
         promote(cv.read().version[0], lce.id)
         new_systems = [
-            make_content_host({
```

Comment:
Should we remove `make_content_host` method at all? I meant not only calls, but its definition